### PR TITLE
adding PVC-->PVS image import and IBM COS docs

### PR DESCRIPTION
**Pull request by iv1111**
This PR includes:

- PowerVC to Power Systems Virtual Server image import documentation
- Cloud Object Storage documentation

### CODE REVIEW

File: managing_providers/_topics/authenticating_to_microsoft_scvmm.md [hunk 1]
```diff
@@ -6,26 +6,26 @@ Microsoft SCVMM servers. You must also set the appropriate execution
 policy on the Microsoft SCVMM server to allow PowerShell scripts from
 the appliance to run remotely.
 
-1.  Log in to the Microsoft SCVMM server.
```
Comment (agrare) [inline managing_providers/_topics/authenticating_to_microsoft_scvmm.md:9]: These changes look unrelated

Comment (iv1111) [reply managing_providers/_topics/authenticating_to_microsoft_scvmm.md:9]: you're right, I will revert it

**Comment by iv1111** [comment]
@agrare removing the WIP

**Review by agrare — APPROVED** [review]
